packages/millicast-sdk/src/Director.js

```diff
@@ -169,6 +169,7 @@ export default class Director {
 
   static async getSubscriber (options, streamAccountId = null, subscriberToken = null) {
     const optionsParsed = getSubscriberOptions(options, streamAccountId, subscriberToken)
+    saveAccountId(optionsParsed.streamAccountId)
```

Comment:
Solution1:

Advice to make 'saveAccountId' function as a method of Logger. So we could invoke it like this:
```suggestion
    Logger.saveAccountId(optionsParsed.streamAccountId)
```

Or in oder to avoid coupling/pollute these with Logger, we could use another class/utility to manage the data . 
```
    const Diagnostics = {
       ........
}
```
We only use Logger to print this Object.


-----------------------------

Solutions2:

A better design using interface.  Director own a property `dataListener` which confirms to an interface like this below
```
  interface DataListener {
    updateAccountId(accountId: string)
   <other functions>
}
```
Then make Logger or Diagnostics implement this interface (like solutions1) and we could register it:
```
   Director.registerDataListener(Logger/Diagnostics)
```
Finally, Director can invoke these callbacks of data listener
```
    // inside of Director methods
    this.dataListener?.updateAccountId(accountId)
```
Note: PeerConnectionStats and Signaling need to have their own DataListener interface as well.

Reply:
I understand this suggestion is meant to look for a clearer way to call these save methods. It is easier to understand that calling `Diagnostics.saveAccountId(accountId)` is meant to save that value for diagnostics purposes.
If that is the case, I think the way to go would be to create a new module named `Diagnostics` (Solution 1), and call those methods from there.
Is that OK?

Reply:
Yes, sounds good.

Reply:
@vincentsong please check the latest changes and lets discuss if that is what you meant.